salt/runc/map.jinja

```diff
@@ -0,0 +1,3 @@
+{%- set runc = salt['grains.filter_by']({
+  'Linux': {}
+}, grain='kernel', merge=salt['pillar.get']('runc')) %}
```

Comment:
Please don't use this hack :).

The correct way to set this is using the "default" argument (https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.grains.html#salt.modules.grains.filter_by)

```
{%- set runc = salt['grains.filter_by']({
  'default': {}
} merge=salt['pillar.get']('runc')) %}
```

Reply:
Oh true it's better :)